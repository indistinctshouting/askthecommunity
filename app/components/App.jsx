const React = require('react');
const axios = require('axios');


// const ReactDOM = require('react-dom');

class App extends React.Component {
  constructor(props) {
    super(props)
    this.state = {
      questions: []
    }
    this.getQuestions = this.getQuestions.bind(this);
  }

  // componentDidMount() {

  // }

  getQuestions(Id) {
    axios.get(`http://localhost:3000/askthecommunity`)
      .then(function (reponse) {
        console.log(response);
      })
      .catch(function (error){
        console.log(error);
      })
  }

  //   //based on the restaurant id: i wanna display all the questions that are stored in the database
  //   //with the according answers
  //   axios.post('/askthecommunity')
  //     .then(function(response) {
  //       console.log(response);
  //     })
  //     .catch(function(error){
  //       console.log(error);
  //     });
  // }

  // getAnswers() {
  //   $.ajax({
  //     type: 'G'
  //   })
  // }
  render() {
    return (
      // <div>
      //   just TAKE IT FROM MY HANDS
   <div>  
    <h1>Question Goes Here</h1>
    <p>Answer will go here</p>
    <button type="button" onClick={() => {alert("Do something!")}}>View question details</button>
    </div>
    )
  }
};

// class App extends React.Component {
//   render() {
//     return <h1>YUHHHH!HH!H</h1>
//   }
// };

module.exports = App; 