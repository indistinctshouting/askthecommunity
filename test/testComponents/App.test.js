import React from 'react';
import {shallow} from 'enzyme'; 
import App from '../../app/components/App'
import etj from 'enzyme-to-json';


it('should render correctly with no props', () => {
  const component = shallow(<App/>);
  
  expect(component).toMatchSnapshot();
});

// const wrapper = shallow(<App />);