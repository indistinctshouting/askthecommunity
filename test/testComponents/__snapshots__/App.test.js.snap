// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly with no props 1`] = `
<div>
  <h1>
    Question Goes Here
  </h1>
  <p>
    Answer will go here
  </p>
  <button
    onClick={[Function]}
    type="button"
  >
    View question details
  </button>
</div>
`;
